xlators/cluster/dht/src/dht-rebalance.c

```diff
@@ -3781,8 +3756,6 @@ gf_defrag_fix_layout(xlator_t *this, gf_defrag_info_t *defrag, loc_t *loc,
                 gf_msg(this->name, GF_LOG_ERROR, 0, DHT_MSG_LAYOUT_FIX_FAILED,
                        "Fix layout failed for %s", entry_loc.path);
 
-                defrag->total_failures++;
```

Comment:
Okay, So you made this change in this patch only. Good.

Reply:
Yes, as I needed to migrate this from gerrit anyway.